src/test/java/shop/CartTest.java

```diff
@@ -0,0 +1,49 @@
+package shop;
+
+import org.junit.jupiter.api.*;
+import org.testng.Assert;
+
+class CartTest {
+
+    RealItem realItem;
+    Cart userCart;
+
+    @BeforeEach
+    public void setUpForTest(){
+        userCart = new Cart("user-cart");
+        realItem = new RealItem();
+        realItem.setName("Pen");
+        realItem.setWeight(6);
+        realItem.setPrice(2);
+        userCart.addRealItem(realItem);
+    }
+
+    @Test
+    @DisplayName("Total price was calculated successfully for 1 Item")
+    public void getTotalPriceForOneItem() {
+        double actualResult = userCart.getTotalPrice();
+        double expectedResult = 2.4;
```

Comment:
Right now it is a magic number. If there is any formula, it will be correct to use it there